src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, Area, Info {

```

Comment:
Don't use abstract classes to set behavior for classes. In this case, it seems that the `Figure` class is being used to define behavior via the `Drawable`, `Area`, and `Info` interfaces. Consider using interfaces to define behavior and use abstract class only when there's a need to share code among different subclasses.